### PR TITLE
Modules

### PR DESCRIPTION
**Pull request by AlejandraKeber**
In this milestone of the project I did the following:
- Setup the repository and add linterns files
- Separate js files using ES6 modules and an index.js file
- Change functions to ES6 arrow functions
- Create a luxon library and add date/time to de js file

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/AlejandraKeber/Awsome-Books-ES6/issues"
+  },
+  "homepage": "https://github.com/AlejandraKeber/Awsome-Books-ES6#readme",
+  "dependencies": {
+    "luxon": "^3.0.4"
+  }
```
Comment (Bondok6) [inline package.json]: - [ ] Great job using `luxon` library 💯, but according to the requirements you should add this package as a `.js` file to the project (in full or minified version), so kindly download [luxon from here](https://moment.github.io/luxon/#/install?id=es6) and save it to your project and import it as module 👌💯



> NOTE: To uninstall "luxon" package from dependencies you can run `npm uninstall luxon` 💡

File: README.md [hunk 2]
```diff
@@ -1,2 +1,48 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (Bondok6) [inline README.md]: - [ ] The Link to MIT.md file is broken, So I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @AlejandraKeber 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No Linter errors ✅
- Correct GitHub flow ✅
- Your repo is professional ✅
- Great job using arrow functions ✅


## Required Changes ♻️
- [ ] _Check the comments under the review._ ⬇

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1,2 +1,48 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (phelian23) [inline README.md]: - [ ] Good job adding the MIT.md file to your project as requested by the previous reviewer, how ever, it is not correctly linked. kindly fix the link.

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @AlejandraKeber,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
No Linter errors ✅
Correct GitHub flow ✅
Your repo is professional ✅

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @AlejandraKeber , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**